src/rpc/rawtransaction.cpp

```diff
@@ -39,6 +39,7 @@
 
 #include <univalue.h>
 
+constexpr static CAmount DEFAULT_MAX_RAW_TX_FEE{COIN / 10};
```

Comment:
Suggest a comment here. Something like:

```
/** High fee for sendrawtransaction and testmempoolaccept. 
 * By default, transaction with a fee higher than this will be rejected by
 * sendrawtransaction and testmempoolaccept. This can be overriden with the
 * maxfeerate argument.
 * */
```

Reply:
I think the release notes and rpc documentation are pretty clear on this. Lets not bloat developers with redundant or excessive documentation

Reply:
Release notes and RPC documentation have nothing to do with code constants.

@promag has already said that this constant name is confusing.

Reply:
promag was worried that there are two constants where one would theoretically be sufficient

Happy to change the name if there is a better one, though.

Reply:
Anyway, changed my mind